server/ntb/commands/sync_topics.py

```diff
@@ -0,0 +1,402 @@
+from os import path
+import sys
+import json
+from datetime import datetime
+import requests
+import socket
+import logging
+
+from flask import current_app as app
+import superdesk
+
+logger = logging.getLogger(__name__)
+RESOURCE_URL = "https://cv.iptc.org/newscodes/mediatopic"
+
+
+class SyncTopicsCommand(superdesk.Command):
+    """Synchronise MediaTopics CV from IPTC
+
+    Downloads that latest Media Topics json from IPTC server,
+    and merges them into the `data/vocabularies.json` file.
```

Comment:
we are using ReST in comments, not markdown, so you must use 2 back-quotes instead of 1 for inline literals:
```rst
`data/vocabularies.json` ⇒ ``data/vocabularies.json``
```

Same think below and in other docstrings.